src/status_im/ui/screens/events.cljs

```diff
@@ -271,6 +272,8 @@
 (register-handler-fx
  :get-fcm-token
  (fn [_ _]
+   ;; XXX: compiled away?
+   (utils/show-popup "TODO" "FCM FX CALLED")
```

Comment:
QUESTION: Expect this to be called but it isn't on release build. The popups in `initialize-account` and `init-wallet` popups are called. Running in debug mode on real device shows this popup. Why?

It is part of the compiled bundle:
```
Oskars-MacBook-Pro:status-react oskarth$ less status.xcarchive/Products/Applications/StatusIm.app/main.jsbundle | tr ',' '\n' | grep "FCM FX" -C 3
status_im.utils.handlers.register_handler_fx.cljs$core$IFn$_invoke$arity$2(cljs.core.cst$kw$get_DASH_fcm_DASH_token
function(e
c){return status_im.utils.utils.show_popup("TODO"
"FCM FX CALLED")
status_im.utils.notifications.get_fcm_token()
cljs.core.PersistentArrayMap.EMPTY})
status_im.components.status_bar={}
```

Reply:
One can see that they should all be called in succession, which they are in debug mode but not for release for whatever reason: https://github.com/status-im/status-react/blob/feature/push-notifications-spike-2-%231599/src/status_im/ui/screens/events.cljs#L168-L170

Reply:
Seems like the reason was too many alerts in quick succession. iOS appears to have a max of two alerts up simultaneously, and which ones depends on delay between them.